.github/workflows/preview.yml

```diff
@@ -7,6 +7,9 @@ on:
     types:
       - completed
 
+env:
+  GH_TOKEN: ${{ github.token }}
```

Comment:
From a security point of view I wonder if we should set this at the top level or not. I'd lean to being as specific as possible. @evgeni any thoughts?

Reply:
That'd mean adding it to every `run` that has `gh`?

Reply:
Can we somehow use CODEOWNERS to prevent changes from non-maintainers in `.github/*` in PRs?

Applied in c4ffa1acaf4ab8951a276ea233df9f8567eac213

Reply:
True. But still a nice angle: PRs that change/introduce GHA that run on PRs and use that token. I can only hope it's similar to GitLab and tied to your user account.

Reply:
> That'd mean adding it to every `run` that has `gh`?

More that any npm package we happen to pull in via `surge` also can access the env var.